tools/utils.ts

```diff
@@ -0,0 +1,71 @@
+import * as yargsParser from 'yargs-parser';
+import type * as Enquirer from 'enquirer';
+
+/**
+ * CLI prompts abstraction to trigger dynamic prompts within a generator
+ *
+ * @remarks
+ * - lazy loads enquirer only when needed making CLI programs faster to load/execute
+ *
+ * @param questions
+ */
+export async function prompt<T extends Record<string, unknown>>(questions: Parameters<Enquirer['prompt']>[0]) {
+  const EnquirerLazy = await await import('enquirer');
```

Comment:
Q: Is the double `await` a mistype?

Reply:
it definitely is. to bad TS doesn't point this out ! 

- we should enable/write a lint rule that will let you know about this issues. Also if you're missing `await` (which is quite crucial and will crash our programs )

Reply:
tracked https://github.com/microsoft/fluentui/issues/19234. TY @andrefcdias 💪

Reply:
Thought it was some new obscure magic I wasn't aware of 😄